README.md

```diff
@@ -579,7 +581,12 @@ oneview_volume 'Volume_1' do
   storage_pool <storage_pool_name>
   volume_template <volume_template_name>
   snapshot_pool <snapshot_pool_name>
-  action [:create, :create_if_missing, :delete]
+  properties <volume_properties>                # Hash. Used in create_from_snapshot action only. Only available on API500 and onwards.
+  device_volume_name <device_volume_name>       # String. Name of the external volume that want to add. Used in add action only. Only available on API500 and onwards.
```

Comment:
Why not to use the name passed to resource? 
Like here: 
```ruby
  oneview_volume 'Volume_1'` do
    ...
  end
```
We can use the name "Volume_1" to the action `add`, right? And we would avoid a new property.

Reply:
Because the ```device_volume_name``` and volume ```name``` are not the same thing. I can import a volume named ```X``` and want this volume named ```Y```.

Reply:
"# String. Name of external volume to be added. Used only in the `add` action. Only available on API500 and onwards."